### PR TITLE
Add healthchecks

### PR DESCRIPTION
**Pull request by rosswhitfield**
https://code.ornl.gov/sns-hfir-scse/infrastructure/web-monitor/-/issues/197

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,23 +8,19 @@ services:
       - "80:80"
     volumes:
       - web-static:/var/www/workflow/static
-      - ./nginx/django.conf:/etc/nginx/conf.d/django.conf
-      - ./nginx/docker-entrypoint.sh:/docker-entrypoint.d/docker-entrypoint.sh
-      - ./nginx/wait-for-it.sh:/usr/bin/wait-for-it.sh
+      - ./nginx/django.conf:/etc/nginx/conf.d/default.conf
+    healthcheck:
+      test: ["CMD", "service", "nginx", "status"]
```
Comment (peterfpeterson) [inline docker-compose.yml:13]: We should copy this back into webref

Comment (rosswhitfield) [reply docker-compose.yml:13]: I copied it from WebRef

File: src/workflow_app/workflow/daemon.py [hunk 2]
```diff
@@ -135,6 +135,24 @@ def restart(self):
         self.stop()
         self.start()
 
+    def status(self):
```
Comment (peterfpeterson) [inline src/workflow_app/workflow/daemon.py:138]: Doesn't psutil do most of this with different interface?

File: docker-compose.yml (hunk 1 above)
Comment (peterfpeterson) [reply docker-compose.yml:13]: I feel so stupid

File: src/workflow_app/workflow/daemon.py (hunk 2 above)
Comment (rosswhitfield) [reply src/workflow_app/workflow/daemon.py:138]: I could replace some of this with `psutil.pid_exists`

**Review by peterfpeterson — APPROVED** [review]

**Pull request merged**